### ISSUE TITLE
custom blacket blooks not working

### ISSUE THREAD

**Issue by deernothere** [TheBeast143/Blooket-Kahoot-Gimkit-Hacks#1]
around a week ago i could use the blacket blooks from this hack, however, it doesnt work anymore. neither does the custom blook url. please fix

**Comment by TheBeast143** [comment]
Hey deernothere, what do you mean by blacket? Did you mean to type blooket or literally https://blacket.org/? Because there is a blanket and a blooket. If you can specify, I can fix the issue.

**Comment by deernothere** [comment]
i saw that there were custom blooks from blacket that you can use in blooket with this, but they dont work and just default to the blue color blook

**Comment by deernothere** [comment]
i dont think this is an issue anymore since the inspect element copy and paste was removed and i dont think you can use custom blooks via the bookmarklet

**Comment by TheBeast143** [comment]
There is a way to bypass the blocked inspect element, tell me if you want me to add it to this repo.

**Comment by TheBeast143** [comment]
Also share this repo with your friends, I need this to get more public :).

**Comment by deernothere** [comment]
alright ill try
